src/server/app.py

```diff
@@ -27,7 +27,9 @@ def submit():
     caption_data = request.get_json()
 
     try:
-        file_name = caption_data["file_name"] + '.json'
+        now = datetime.now()
+        date_time = now.strftime("%m.%d.%Y_%H-%M-%S")
```

Comment:
I see what Marko was saying about the time not being the same. If you wouldn't mind, could you rename date_time to time_stamp and pass it into consume, and then use time_stamp in there instead of date_time.